docs/markdown/Builtin-options.md

```diff
@@ -267,6 +267,16 @@ Since 0.56.0 `warning_level` can also be defined per subproject.
 Some Meson modules have built-in options. They can be set by prefixing the option
 name with the module name: `-D<module>.<option>=<value>` (e.g. `-Dpython.platlibdir=/foo`).
 
+### Gnome module
+
+| Option           | Default value | Possible values          | Description |
+| ------           | ------------- | ------------------------ | ----------- |
+| gtkdoc           | enabled       | enabled, disabled, auto  | Whether to build the Gtk API html docs (Since 0.61.0) |
```

Comment:
I think it should be `auto` by default, so it can be controlled by `-Dauto_features=disabled`.

Reply:
I don't really have a strong opinion on that, but my rationale was:

Currently there is no option, and it is required.
Making it auto by default is a behavior change.

Is this behavior change actually problematic in any way? You tell me... :D